gradle.properties

```diff
@@ -1,14 +1,26 @@
-org.gradle.jvmargs=-Xmx4g
+org.gradle.caching=true
+org.gradle.configureondemand=true
+org.gradle.jvmargs=-Xmx4g -Dfile.encoding=UTF-8
+org.gradle.parallel=true
 
 # AndroidX
 android.useAndroidX=true
 
+# Enable install debug apk manually
+android.injected.testOnly=false
+
 # Disable welcome message.
 systemProp.org.gradle.internal.launcher.welcomeMessageEnabled=false
 
 # Disable adding the standard library by default.
 kotlin.stdlib.default.dependency=false
 
+# Enhance speed of kotlin build
+kotlin.code.style=official
+kotlin.incremental=true
```

Comment:
These are all enabled by default.

Reply:
![image](https://user-images.githubusercontent.com/10363352/110059502-4ec90100-7d9f-11eb-8390-ae411cb0860b.png)
Default value is false

Reply:
Hmm according to [this blog](https://blog.jetbrains.com/kotlin/2017/03/kotlin-1-1-1-is-out/) it was enabled by default in Kotlin `1.1.1`.

Reply:
Ok, removed